notification-api-service/src/main/java/com/smarsh/notificationservice/api/util/PathUtil.java

```diff
@@ -9,22 +9,24 @@
 public class PathUtil {
 
     private PathUtil() {
+        throw new RuntimeException();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://internal-fake-sonarqube-elb-2-1800924245.us-west-2.elb.amazonaws.com/coding_rules#rule_key=squid%3AS00112)